envoy/requirements.in

```diff
@@ -0,0 +1 @@
+requests==2.18.4
```

Comment:
you need to generate the requirements.txt through the general mechanism we use for this: `pip-compile --generate-hashes --output-file requirements.txt requirements.in`

this is a requirement conflict with datadog-agent-base and a lot of other checks. Please use the version we're already using. If you want to upgrade, you can do that in another PR as it requires a good bit of work to ensure there aren't any conflicts.